src/current/_includes/v23.1/sql/privileges.md

```diff
@@ -1,28 +1,28 @@
-Privilege | Levels
-----------|------------
-`ALL` | Database, Schema, Table, Sequence, Type
-`BACKUP` | System, Database, Table
-`CANCELQUERY` | System
-`CHANGEFEED` | Table
-`CONNECT` | Database
-`CREATE` | Database, Schema, Table, Sequence
-`DELETE` | Table, Sequence
-`DROP` | Database, Table, Sequence
-`EXECUTE` | Function
-`EXTERNALCONNECTION` | System
-`EXTERNALIOIMPLICITACCESS` | System
-`INSERT` | Table, Sequence
-`MODIFYCLUSTERSETTING` | System
-`MODIFYSQLCLUSTERSETTING` | System
-`NOSQLLOGIN` | System
-`RESTORE` | System, Database
-`SELECT` | Table, Sequence
-`UPDATE` | Table, Sequence
-`USAGE`  | Function, Schema, Sequence, Type
-`VIEWACTIVITY` | System
-`VIEWACTIVITYREDACTED` | System
-`VIEWCLUSTERMETADATA` | System
-`VIEWCLUSTERSETTING` | System
-`VIEWDEBUG` | System
-`VIEWJOB` | System
-`ZONECONFIG` | Database, Table, Sequence
+Privilege | Levels | Description
+----------|--------|------------
+`ALL` | System, Database, Schema, Table, Sequence, Type | Grants all privileges at the system, database, schema, table, sequence, or type level.
+`BACKUP` | System, Database, Table | Grants the ability to create [backups]({% link {{ page.version.version }}/backup-and-restore-overview.md %}) at the system, database, or table level.
+`CANCELQUERY` | System | Grants the ability to cancel queries.
+`CHANGEFEED` | Table | Grants the ability to create [changefeeds]({% link {{ page.version.version }}/change-data-capture-overview.md %}) on a table.
+`CONNECT` | Database | Grants the ability to connect to the database.
+`CREATE` | Database, Schema, Table, Sequence | Grants the ability to create objects at the database, schema, table, or sequence level.
```

Comment:
Perhaps we should add a bit here that says having CREATE also grants the ability to change multi region zone configs. Regardless of whether ZONECONFIG is granted or not.

Reply:
Thoughts @rafiss ?

Reply:
that sounds good. you mean CREATE at the database level right?

Reply:
+1, I think its good to point out that CREATE allows access to ZONECONFIG.

Reply:
Added. And @data-matt I'll need your approval to get this merged, as you requested changes.